### PR TITLE
Collapsed room list has unaligned room tiles #4030 version 2

### PR DESCRIPTION
**Pull request by TestardR**
@turt2live I have done a clean pull request (deleted previous commits).

@turt2live raised the issue.

I solved it passing "collapsed" as a prop from RoomTile to InviteOnlyIcon. Then on, I offered a different css style if collapsed returned true or false.

The new style looks like this:

Signed-off-by: Romain Testard romain.rtestard@gmail.com

Fixes https://github.com/vector-im/riot-web/issues/12171

### CODE REVIEW

**Comment by turt2live** [comment]
@nadonomy can you take a look at this for design/product review please? Thanks!

File: res/css/views/rooms/_InviteOnlyIcon.scss [hunk 1]
```diff
@@ -14,27 +14,45 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-.mx_InviteOnlyIcon {
+@define-mixin mx_InviteOnlyIcon {
+  width: 12px;
```
Comment (turt2live) [inline res/css/views/rooms/_InviteOnlyIcon.scss]: all of our CSS is supposed to be 4 spaces indented.

File: src/components/views/rooms/InviteOnlyIcon.js [hunk 2]
```diff
@@ -46,7 +48,7 @@ export default class InviteOnlyIcon extends React.Component {
         if (this.state.hover) {
             tooltip = <Tooltip className="mx_InviteOnlyIcon_tooltip" label={_t("Invite only")} dir="auto" />;
         }
-        return (<div className="mx_InviteOnlyIcon"
+        return (<div className={style}
```
Comment (turt2live) [inline src/components/views/rooms/InviteOnlyIcon.js]: preferably we call the variable something like `className` or `classes`

**Comment by TestardR** [comment]
@turt2live Changes were made according to your recommendations.

**Comment by nadonomy** [comment]
Hey @TestardR thanks for this!

To review— can you post a full screen screenshot of the entire app UI please? Ideally one with and without a full user list open.

**Comment by jryans** [comment]
I have started CI in the background; we should be ready to merge once it completes.

**Pull request merged**